app/src/main/java/com/example/myweatherapp/utils/WeatherUtils.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.myweatherapp.utils
+
+import java.text.SimpleDateFormat
+import java.util.Locale
```

Comment:
create here a StringUtils file or something, do not keep it global pls

Reply:
Done thx